sssd_test_framework/topology.py

```diff
@@ -118,6 +119,17 @@ def test_ldap(client: Client, ldap: LDAP):
     .. topology-mark:: KnownTopology.IPATrustSamba
     """
 
+    IPATrustIPA = SSSDTopologyMark(
+        name="ipa-trust-ipa",
+        topology=Topology(TopologyDomain("sssd", client=1, ipa=2)),
```

Comment:
Second IPA should go into different domain.

If you put two ore more identical roles inside single domain, it should mean that the hosts are identical (i.e. ipa replica).

Reply:
Updated, along with sssd tests mhc.yaml.